poetry.lock

```diff
@@ -10,11 +10,14 @@ python-versions = ">=3.8"
 beautifulsoup4 = "*"
 decorator = "*"
 distro = {version = "*", markers = "sys_platform != \"darwin\" and sys_platform != \"win32\""}
-orjson = "*"
+orjson = {version = "*", markers = "platform_machine == \"x86_64\""}
```

Comment:
Thanks to your broad knowledge and advice,I manage to use poetry. Is this line correct?